konch.py

```diff
@@ -345,11 +345,16 @@ def check_availability(self):
 
     def start(self):
         try:
-            from ptpython.repl import embed
+            from ptpython.repl import embed, run_config
         except ImportError:
             raise ShellNotAvailableError("PtPython shell not available.")
         print(self.banner)
-        embed(globals=self.context, vi_mode=self.ptpy_vi_mode)
+
+        if not os.path.isfile(os.path.expanduser("~/.ptpython/config.py")):
+            run_config = None
```

Comment:
In general, it's good to avoid re-assigning imported symbols (this is why flake8 is failing). See my alternative approach in my other comment.